components/eventing-controller/api/v1alpha1/subscription_types_unit_test.go

```diff
@@ -7,6 +8,8 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/kyma-project/kyma/components/eventing-controller/pkg/env"
+
+	"github.com/kyma-project/kyma/components/eventing-controller/api/v1alpha1"
 )
 
 const (
```

Comment:
the `orderCreatedEventType` is redeclared in the same package, can you please remove it. 

Ideally you could move some common files into the new `fixtures_test` file.

Reply:
working on the tests in parallel is terrible.